MediaGalleryUi/Test/Mftf/Section/AdminEnhancedMediaGalleryFiltersSection.xml

```diff
@@ -13,5 +13,8 @@
         <element name="applyFilters" type="button" selector="//div[@class='media-gallery-container']//button[@data-action='grid-filter-apply']"/>
         <element name="activeFilter" type="text" selector="//div[@class='media-gallery-container']//div[@class='admin__current-filters-list-wrap']//span[contains( ., '{{filter}}')]" parameterized="true"/>
         <element name="activeFilterPlaceholder" type="text" selector="//div[@class='media-gallery-container']//div[@class='admin__current-filters-list-wrap']"/>
+        <element name="usedInSelectDropdown" type="text" selector="//div[@class='admin__action-multiselect-text' and text()='Select...']"/>
```

Comment:
The selectors should be more specific, we are going to have more than one multiselect in the filters section

Reply:
ok